src/components/Hello.vue

```diff
@@ -10,8 +31,19 @@
   @Component
   export default class Hello extends Vue {
     name: string = 'app'
-    msg: string = 'Welcome to Your Vue.js App'
+    msg: string = 'Welcome to Your Vue.js App!!'
+    seen: boolean = false;
+    clicked() : void {
+      var interest = <HTMLInputElement> document.getElementById("interest"); 
```

Comment:
This code works but would be cleaner using [reactive](https://en.wikipedia.org/wiki/Reactive_programming)-style instead of imperative-style programming. Give it a shot, and can chat more or do some peer-programming.